app/_src/gateway-operator/guides/hardening/control-plane-watch-namespaces.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Limiting namespaces watched by ControlPlane
+---
+
+{{site.kgo_product_name}}'s `ControlPlane` watches all namespaces by default.
+This is a convenient out of the box experience but might not fit all production environments,
+specifically those with multiple teams sharing the same cluster or in multitenant environments.
+
+To limit the namespaces watched by `ControlPlane`, you can set the `watchNamespaces` field in the `ControlPlane`'s `spec`.
+
+## ControlPlane's watchNamespaces field
+
+The `spec.watchNamespaces.type` accepts 3 values to control this behavior:
+
+- `all`: default, as the name implies, watch resources in all namespaces
+- `own`: makes the `ControlPlane` only watch resources in its own namespace
+- `list`: makes the `ControlPlane` watch resources in its own namespace and the provided list of namespaces
+  Using "list" also adds ControlPlane's own namespace to the list of watched namespaces
+  because that's that KIC does.
+  The reason for this is the publish service (`DataPlane`'s `Service` exposed by {{site.base_gateway}})
+  by default would exist in the same namespace as `ControlPlane`.
+
+> Note: Please mark that this setting in `ControlPlane` will set the `CONTROLLER_WATCH_NAMESPACE`
+> environment variable in managed {{ site.kic_product_name }} so setting this via
```

Comment:
Does this mean setting `CONTROLLER_WATCH_NAMESPACE` via `podTemplateSpec` has higher priority than this? I think the word here is not very clear. How about 
```suggestion
> environment variable in managed {{ site.kic_product_name }} so setting the watched namespaces in the environment variable via
```